### PR TITLE
Update README, tags and description

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
-# ASPxGridView - A simple Batch Editing implementation
+# Grid View for ASP.NET Web Forms - A simple batch editing implementation
+
 <!-- run online -->
+
 **[[Run Online]](https://codecentral.devexpress.com/e5045/)**
+
 <!-- run online end -->
 
+This example demonstrates how to use the [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control's batch edit mode events to update a custom object data source stored in session.
```
Comment (dirk-pieterse) [inline Readme.md]: `in session` a session? the session?

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,46 @@
...
+
+![Simple Batch Edit implementation](simple-batch-edit.png)
+
+When a Grid View is bound to an object data source at runtime, you need to implement CRUD logic required to edit data. To achieve this, use one of the following options:
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
When a Grid View is bound to an object data source at runtime, you need to implement the CRUD logic required to edit data. To do this, use one of the following options:
```

File: Readme.md [hunk 3]
```diff
@@ -1,24 +1,46 @@
...
+- Handle the ASPxGridView's [RowInserting](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.RowInserting), [RowUpdating](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.RowUpdating), and [RowDeleting](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.RowDeleting) events.
+
+  This technique allows you to handle each CRUD operation type individually in a separate event handler.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
  This technique allows you to handle each CRUD operation type in a separate event handler.
```

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply Readme.md]: Changed to "in the session"

**Pull request merged**